daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/ledger/BlindingTransaction.scala

```diff
@@ -58,48 +58,53 @@ object BlindingTransaction {
         parentExerciseWitnesses: Set[Party],
         nodeId: NodeId,
     ): BlindState = {
-      val action = tx.nodes.get(nodeId) match {
+      tx.nodes.get(nodeId) match {
```

Comment:
Sorry I shouldn’t be sneaking this in here. Will try to spin it out into a separate PR with tests.

Reply:
https://github.com/digital-asset/daml/pull/9431